aggregator/graphqlResponseGetter.go

```diff
@@ -11,7 +12,7 @@ import (
 
 // GraphqlResponseGetter wraps over the default http client
 type GraphqlResponseGetter struct {
-	authentication.AuthClient
+	AuthClient authentication.AuthClient
```

Comment:
still, why do we have GraphqlResponseGetter struct exported?
also, this is a dangerous business to have a struct with and exported item like AuthClient that also contains business logic. I can make your code panic on L27 by setting from exterior AutcClient to nil.

Reply:
done, I made also httpResponseGetter unexported for consistency

Reply:
not done, still risky having AuthClient exported